spirv_msl.hpp

```diff
@@ -69,6 +69,7 @@ struct MSLShaderInterfaceVariable
 	MSLShaderVariableFormat format = MSL_SHADER_VARIABLE_FORMAT_OTHER;
 	spv::BuiltIn builtin = spv::BuiltInMax;
 	uint32_t vecsize = 0;
+	bool patch = false;
```

Comment:
This should be an enum to allow for mesh shaders at some point.

Reply:
Done.